example-mlem-get-started/code/src/train.py

```diff
@@ -0,0 +1,23 @@
+from mlem.api import save
+from sklearn.datasets import load_iris
+from sklearn.ensemble import RandomForestClassifier
+
+
+def main():
+    data, y = load_iris(return_X_y=True, as_frame=True)
+    rf = RandomForestClassifier(
+        n_jobs=2,
+        random_state=42,
+    )
+    rf.fit(data, y)
+
+    save(
+        rf,
```

Comment:
should we call it `model` for simplicity?

Reply:
Don't think it matter that much. And we have docs in place, that rely on this repo, so let's not change this.